com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -89,45 +89,153 @@ public static Texture2D CreateColoredTexture2D(Color color, string textureName)
-        public static void DrawFixMeBox(GUIContent message, Action action)
+        public static void DrawFixMeBox(string message, MessageType messageType, Action action)
         {
-            GUILayout.Space(2);
-            using (new EditorGUILayout.HorizontalScope(EditorStyles.helpBox))
-            {
-                EditorGUILayout.LabelField(message, CoreEditorStyles.helpBoxLabelStyle);
-                GUILayout.FlexibleSpace();
-                using (new EditorGUILayout.VerticalScope())
-                {
-                    GUILayout.FlexibleSpace();
+            DrawFixMeBox(EditorGUIUtility.TrTextContentWithIcon(message, CoreEditorStyles.GetMessageTypeIcon(messageType)), "Fix", action);
+        }
 
-                    if (GUILayout.Button("Fix", GUILayout.Width(60)))
-                        action();
+        /// <summary>Draw a help box with the Fix button.</summary>
+        /// <param name="message">The message text.</param>
+        /// <param name="messageType">The type of the message.</param>
+        /// <param name="buttonLabel">The button text.</param>
+        /// <param name="action">When the user clicks the button, Unity performs this action.</param>
+        public static void DrawFixMeBox(string message, MessageType messageType, string buttonLabel, Action action)
+        {
+            DrawFixMeBox(EditorGUIUtility.TrTextContentWithIcon(message, CoreEditorStyles.GetMessageTypeIcon(messageType)), buttonLabel, action);
+        }
 
-                    GUILayout.FlexibleSpace();
-                }
-            }
-            GUILayout.Space(5);
+        /// <summary>Draw a help box with the Fix button.</summary>
+        /// <param name="message">The message with icon if need.</param>
+        /// <param name="action">When the user clicks the button, Unity performs this action.</param>
+        public static void DrawFixMeBox(GUIContent message, Action action)
+        {
+            DrawFixMeBox(message, "Fix", action);
         }
 
         /// <summary>Draw a help box with the Fix button.</summary>
-        /// <param name="text">The message text.</param>
-        /// <param name="messageType">The type of the message.</param>
+        /// <param name="message">The message with icon if need.</param>
+        /// <param name="buttonLabel">The button text.</param>
         /// <param name="action">When the user clicks the button, Unity performs this action.</param>
-        public static void DrawFixMeBox(string text, MessageType messageType, Action action)
+        public static void DrawFixMeBox(GUIContent message, string buttonLabel, Action action)
         {
-            DrawFixMeBox(EditorGUIUtility.TrTextContentWithIcon(text, CoreEditorStyles.GetMessageTypeIcon(messageType)), action);
+            EditorGUILayout.BeginHorizontal();
+
+            float indent = EditorGUI.indentLevel * 15 - EditorStyles.helpBox.margin.left;
```

Comment:
what is this 15? 
We tend to try to not have magic numbers but make a variable out of them and name them properly, like const  k_IndentMargin =15; Or something similar.